lib/services/classes.js

```diff
@@ -85,9 +85,27 @@ module.exports = class classesService extends Schmervice.Service {
   }
 
   async createClass(newClassData, role = null) {
-    const { Classes, UserRole } = this.server.models();
+    const { Classes, UserRole, PathwayCourses } = this.server.models();
     let newClass;
 
+    try {
+      // setting default max_enrolment to 10 students for spoken english classes
+      const pathwayName = await PathwayCourses.query()
+        .where('course_id', newClassData[0].course_id)
+        .withGraphFetched('pathway')
+        .modifyGraph('pathway', (builder) => builder.select('name'));
+
+      if (pathwayName.length > 0 && pathwayName[0].pathway.name == 'Spoken English') {
+        newClassData.forEach((class_) => {
+          if (!class_.max_enrolment) {
+            class_.max_enrolment = 10;
+          }
+        });
```

Comment:
Can also do default assignments in the following way, although it may be a little less readable:
```suggestion
        newClassData.forEach((class_) => class_.max_enrolment ||= 10);
```
The `||=` in this case is just syntactic sugar for `class_.max_enrolment = class_.max_enrolment || 10` so an alternate suggestion that might be a little more readable would be to write it out fully like that.